libbeat/template/template.go

```diff
@@ -17,14 +17,15 @@ var (
 )
 
 type Template struct {
-	index       string
-	beatVersion Version
-	esVersion   Version
-	settings    common.MapStr
+	index          string
+	beatVersion    Version
+	esVersion      Version
+	settings       common.MapStr
```

Comment:
Seeing both names I was thinking if it could be `settings._source` and `settings.x` but I would not know how to call `x` :-(

Is there are stuff we potentially going to add here?